docs/get-started/ClickHouse.md

```diff
@@ -670,24 +678,6 @@ spark.celeborn.storage.hdfs.dir hdfs://<namenode>/celeborn
 spark.dynamicAllocation.enabled false
 ```
 
-#### Celeborn Columnar Shuffle Support
-Currently, the supported Celeborn versions are `0.3.x` and `0.4.0`.
-The native Celeborn support can be enabled by the following configuration
-```
-spark.shuffle.manager=org.apache.spark.shuffle.gluten.celeborn.CelebornShuffleManager
-```
-
-quickly start a celeborn cluster
```

Comment:
why  this  section is deleted ?

Reply:
@taiyang-li, the `get-started` of velox does not mention how to quickly start a celeborn cluster. Meanwhile, this section could let users follow the quick start of celeborn, which does not need to mention here.